server/game/cards/11.3-SoKL/DrownedGodFanatic.js

```diff
@@ -7,7 +7,7 @@ class DrownedGodFanatic extends DrawCard {
             title: 'Put Drowned God Fanatic into play',
             condition: () => this.controller.canPutIntoPlay(this),
             location: 'dead pile',
-            cost: ability.costs.kill(),
+            cost: ability.costs.kill(card => card.getType() === 'character'),
```

Comment:
This should be fixed by modifying `/server/game/costs/KillCost.js` instead, since you cannot ever kill locations. Otherwise, we'll need to specify this on every single usage of the kill cost in the future.

Reply:
*nod*.

FWIW I did ask about that on discord/IRC, but sent the PR this way as it's consistent with how the only other card needing this (Given to the Drowned God) is currently doing.

I'll update the PR to fix `KillCost`.